imdclient/IMDClient.py

```diff
@@ -112,13 +120,15 @@ def get_imdframe(self):
             except EOFError:
                 # in this case, consumer is already finished
                 # and doesn't need to be notified
+                # self._wait(self._wait_after_disconnect)
```

Comment:
Remove commented code

Reply:
Also, IMO there is no harm in still attempting to send this packet even if we are pretty sure the simulation is over. It's possible the simulation simply is going too slow and the client thinks it is over, but it actually isn't